docs/core/install/linux-package-manager-fedora31.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Install .NET Core on Fedora 31 - package manager - .NET Core
+description: Use a package manager to install .NET Core SDK and runtime on Fedora 31.
+author: thraka
+ms.author: adegeo
+ms.date: 12/17/2019
+---
+
+# Fedora 31 Package Manager - Install .NET Core
+
+[!INCLUDE [package-manager-switcher](./includes/package-manager-switcher.md)]
+
+This article describes how to use a package manager to install .NET Core on Fedora 31. If you're installing the runtime, we suggest you install the [ASP.NET Core runtime](#install-the-aspnet-core-runtime), as it includes both .NET Core and ASP.NET Core runtimes.
+
+## Register Microsoft key and feed
+
+Before installing .NET, you'll need to:
+
+- Register the Microsoft key
+- register the product repository
+- Install required dependencies
```

Comment:
```suggestion
- Install required dependencies.
```

Reply:
For single sentence lists, you don't need a . according to the MS Style Guide (I just learned that.)

Reply:
@Thraka, Hmm, I see the example in the style-guide does the opposite.

https://docs.microsoft.com/en-us/style-guide/scannable-content/lists#bulleted-lists

Reply:
@Youssef1313's link is what I use as a reference for lists. Can you point to where on the style guide says we're not supposed to add periods @Thraka? The exception would be if there are 3 words or less (https://docs.microsoft.com/en-us/style-guide/punctuation/periods).

Reply:
https://styleguides.azurewebsites.net/Styleguide/Read?id=2696&topicid=28112

But perhaps I was misinterpreting it.

Reply:
Yeah, usually more specific style guides can override the Microsoft style guide. I think the key here is to determine whether these are complete sentences or not.

Reply:
I just merged a PR that fixes all of the other files too.